mautic/mautic
# Config import dir

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Automated tests included? | No
| Related user documentation PR URL | tooltip provided for new setting
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | This allows the import directory to be configured
| BC breaks? | No
| Deprecations? | None

#### Description:
This is a small modification to paths_helper which allows the import directory to be configured. It also limits import "Done" notifications from the import command to only fire when the file is in some final state (Failed, Completed, Stopped)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Edit the configuration->System->General and update the import dir
3. check app/config/local.php to insure import_dir is defined
4. Check the filesystem to insure the directory was created
